boa3_test/tests/boatestcase.py

```diff
@@ -294,6 +298,27 @@ async def deploy(
 
             return cls.deployed_contracts[_manifest.name]
 
+    @classmethod
+    async def get_storage(
+            cls,
+            prefix: Optional[bytes] = None,
+            *,
+            target_contract: Optional[types.UInt160] = None,
+            remove_prefix: bool = False,
+            key_post_processor: Optional[PostProcessor] = None,
+            values_post_processor: Optional[PostProcessor] = None,
```

Comment:
I think it would be nice to have a description of what the parameters are. It might not be too obvious what some of the Optional parameters represent

Reply:
This method is from boa test constructor, I didn't want to overwrite their docstring